sdcm/cluster_k8s/__init__.py

```diff
@@ -1036,17 +1036,20 @@ def deploy_ingress_controller(self, pool_name: str = None):
                 memory_limit = convert_memory_units_to_k8s_value(memory_limit)
                 cpu_request = convert_cpu_units_to_k8s_value(cpu_request)
                 memory_request = convert_memory_units_to_k8s_value(memory_request)
+                haproxy_maxconn = 70000
             else:
-                cpu_limit = '100m'
-                memory_limit = '50M'
+                cpu_limit = '400m'
+                memory_limit = '200M'
                 cpu_request = '100m'
                 memory_request = '50M'
+                haproxy_maxconn = 0
```

Comment:
Why `0`?

Reply:
that's the default, and would make it to calculate it on it's own

Reply:
Please, add a comment above this line saying it will trigger an autocalculation.
Because it is unclear and not only I may have this question.